src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -863,7 +863,7 @@ private function loadDeleteForm(): void
         $this->template->assign('deleteForm', $deleteForm->createView());
     }
 
-    private function getHiddenJsonField(string $name, string $json): SpoonFormHidden
+    private function getHiddenJsonField(string $name, ?string $json = null): SpoonFormHidden
```

Comment:
i never said that... It gets called with 2 params, it doesn't need to. Whatever I'll change it